packages/sit-onyx/src/composables/useAutoFocus.ts

```diff
@@ -0,0 +1,12 @@
+import { onMounted, type Ref } from "vue";
+
+export const useAutofocus = (
+  inputRef: Ref<HTMLElement | null>,
+  props: { autofocus: boolean; loading?: boolean },
+) => {
+  onMounted(() => {
+    if (props.autofocus && !(props.loading ?? false) && document.activeElement !== inputRef.value) {
```

Comment:
It is fine to directly use `!props.loading` here, as it will convert nullish values to false
```suggestion
    if (props.autofocus && !props.loading && document.activeElement !== inputRef.value) {
```